6_temp_coop_fetch.yml

```diff
@@ -18,8 +18,8 @@ targets:
   6_temp_coop_fetch:
     depends:
       - 6_temp_coop_fetch/downloads/long_term_ntl.csv.ind
-      - 6_temp_coop_fetch/downloads/mendota_daily_buoy.csv.ind
-      - 6_temp_coop_fetch/downloads/mendota_temps_long.csv.ind
+      # THIS IS FAILING TO DOWNLOAD - 6_temp_coop_fetch/downloads/mendota_daily_buoy.csv.ind
+      # THIS IS FAILING TO DOWNLOAD - - 6_temp_coop_fetch/downloads/mendota_temps_long.csv.ind
```

Comment:
hmm, I'm able to browse to https://lter.limnology.wisc.edu/sites/default/files/data/ntl130_1_v5.csv, https://lter.limnology.wisc.edu/sites/default/files/data/ntl130_1_v7.csv, and https://lter.limnology.wisc.edu/sites/default/files/data/Lake_Mendota_temps_long.csv. At the moment. Is it possible the site was just down temporarily?

Reply:
It is odd. I noticed this in the LAGOS fetches too - the pasta (the underlying system that EDI/LTER uses) doesn't seem to work with `download.file` or `GET` for these files, even though the browser downloads work. I didn't mess around with trying to figure out why and I also think these files should be in `in` instead of a web fetch here.